### DISCUSSION TITLE
Python object detection for custom apps is not deploying successfully.

### DISCUSSION THREAD

**Discussion by vishalkumarbit** [AxisCommunications/discussions#194] [ACAP] [answered]
I'm attempting to implement a computer vision example from the Axis Communication GitHub repo https://github.com/AxisCommunications/acap-computer-vision-sdk-examples/tree/main/object-detector-python. We are unable to track an error that this custom app is throwing on logs(TLS handshake error: EOF). Please let us know how to resolve this problem as soon as possible.

**Comment by daniel-falk** [comment, marked as answer]
@vishalkumarbit Can you share more context of the error logs? Is it actually the application that logs the message, not the docker daemon during the installation? As far as I can see there should be no TLS involved when running the application.

If it is during install of the application you can disable TLS for the docker daemon API by setting `root.dockerdwrapper.UseTLS` to false and not using `--tlsverify` in the docker command, see [more here](https://github.com/AxisCommunications/docker-acap).

You can also take a look at my step-by-step videos getting the pose detector up and running, there are also some tips for debugging:
[Deep learning in AXIS cameras part 1: Getting started](https://youtu.be/5sAVn1ayA04)
[Deep learning in AXIS cameras part 2: Prototyping and debugging of TFServing, Larod and Docker](https://youtu.be/2te9yRPEPR0)
